bundles/automation/org.eclipse.smarthome.automation.module.script/ESH-INF/automation/moduletypes/ScriptTypes.json

```diff
@@ -43,7 +43,8 @@
                         "label": "Javascript",
                         "value": "application/javascript"
                     }
-                ]
+                ],
+                "defaultValue":"application/javascript"
```

Comment:
Yes, at least as long as [this](https://github.com/eclipse/smarthome/issues/2819#issuecomment-272825332) isn't solved yet. "Value" will have to be removed with https://github.com/eclipse/smarthome/issues/2842 again.